### ISSUE TITLE
deb/RPM changelogs should be based on release notes

### ISSUE THREAD

**Issue by fasaxc** [projectcalico/felix#1393]
Currently they're auto-generated from git history, i think we should be using the tag release notes, which are hand-edited as part of the release process so they tend to be a lot nicer/more concise.

@neiljerram would that be easy to do?

**Comment by neiljerram** [comment]
They are already generated from the tag notes: https://github.com/projectcalico/felix/blob/master/utils/make-packages.sh#L68-L71

**Comment by fasaxc** [comment]
Ah, maybe the issue if that there were multiple tags and one of the earlier tags had a messy note attached, maybe we should give the option to edit the note?

**Comment by neiljerram** [comment]
Yes, the messy note was the problem.  The option to edit is given at that time of creating that note - and should be used! - but yes I suppose we could add another opportunity when doing the packaging.

**Comment by fasaxc** [comment]
I think this came down to doing several tags before cutting one deb/rpm release.  The most recent tag was the GA 2.1.0 tag so we put extra effort into crafting that tag note but the earlier ones were interim pre-releases, which had messy tag notes.  Also, when I cut 2.1.0, I combined the changelogs from all the pre-releases to give a complete picture rather than a delta to the most recent RC so we ended up with duplication.
